test/tests/gcc-c-hello-world/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+image="$1"
+dirTest="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+dirContainer='/usr/src/c'
+
+docker run --rm -v "$dirTest":"$dirContainer":rw -w "$dirContainer" "$image" gcc -o hello-world  hello-world.c
+"$dirTest"/hello-world
+rm -rf "$dirTest"/hello-world
```

Comment:
I was testing your suggestion, but the `rw` is needed, because the executable have to be created. Error:  `cannot open output file hello-world: Read-only file system`